sdk/storage/Azure.Storage.Files.DataLake/src/DataLakeFileClient.cs

```diff
@@ -2827,6 +2827,114 @@ internal virtual async Task<Response<PathInfo>> FlushInternal(
         #endregion
 
         #region Read Data
+        /// <summary>
```

Comment:
Super nitpicky but it looks from this diff like these are a bit intermingled with the deprecated read APIs. I'd much prefer these new ones entirely separate from the old ones so they can be maintained together and not having to scroll through old deprecated code. Ideally, the old read implementations could even be wrapped in a `#region Deprecated` within the existing `#region Read Data`, that way we don't have to scroll through them all the time.

Also, traditionally sync and async versions of APIs are immediately adjacent to each other in C#.

I would imagine an ordered layout as follows, but don't feel a need to follow this exactly, especially if it involves moving tons of older code around.

- region Read Data
  - region Deprecated
    - all the old read methods
  - ReadStreaming()
  - ReadStreamingAsync()
  - ReadContent()
  - ReadContentAsync()

Reply:
Good callout, like this idea